### ISSUE TITLE
[SF][A11y][ui5-table] Data tables missing column header association with JAWS

### ISSUE THREAD

**Issue by 2bno1** [SAP/ui5-webcomponents#4702]
### **Bug Description**
Using JAWS, tab to the table, switch to VPC mode, press CNTL+ALT+arrow keys and navigate through the cells -> observe the column header association is not made. (Note: I have confirmed it works with NVDA and problem is specific to JAWS. However, since JAWS is our primary screen reader and the screenshots below show that the SAP UI5 responsive tables work just fine with column header association with various versions of JAWS.)
Issue exists in WC data table
![WC_table_column_header_association_missing](https://user-images.githubusercontent.com/1751846/152716653-b39b0559-cf2c-4927-9958-f9f352a4a92e.png)

### **Expected Behavior**
Column header association must be made when cells are read in VPC mode.
UI5 examples where column header association works just fine in VPC mode with various versions of JAWS
![col_header_association](https://user-images.githubusercontent.com/1751846/152716768-9b233197-2eef-4bbe-89af-883dab0d4300.png)

### **Context**
- UI5 Web Components version: 1.0.2
- OS/Platform: Windows 10
- Browser: Chrome
- Affected component: ui5-table

The priority indicates the severity of the issue. To set the appropriate priority consider the following criteria:
- Breaks entire application or system - High or Very High
- Accessibility issue - Medium or High
- Functional issue - Medium or High
- Visual issue - Low or Medium

**Note:** The priority might be re-evaluated by the issue processor.

**Comment by alexandar-mitsev** [comment]
Hi @2bno1 ,

Can you please give more details of where and how you reproduce. Which version of Jaws are you using, which exact version of UI5 web components and what is the code of the table you are testing.

I am testing on the demo page for the table in latest version of UI5 Web Components - https://sap.github.io/ui5-webcomponents/playground/components/Table/. With Jaws 2021. And the result is correct - the titles of the columns are read out - check the attachment below.
![image](https://user-images.githubusercontent.com/2347042/152826538-5a7d83f8-803e-475a-8287-1a463b5a67e3.png)

Can you confirm that you are experiencing the issue with the latest version?

Thanks and Regards,
Alexandar Mitsev

**Comment by anuvenkatesh1** [comment]
@alexandar-mitsev I'm from Successfactors accessibility team
It appears you have read the table in "forms" mode - ie once you landed on the table you used arrow keys to go row by row. 
In VPC/Browse mode, you have to press CNTL+ALT+arrow keys to read cell by cell. It's another way a screen reader user will read the cells of the data table. 
I have tested with both JAWS 2020 and JAWS 2022 and I observe the same result - as I navigate through the cells, the column header association is missing. 
![JAWS_2022_browse](https://user-images.githubusercontent.com/32471459/152844172-d2e6f05e-c761-4ede-96ab-6839368dfc23.png)

Comparison with UI5 responsive table in Browse mode - column header association is made as we navigate through the cells using CNTL+ALT+Arrow keys

https://user-images.githubusercontent.com/32471459/152844308-0702188e-08f5-4b27-82fe-67b3adc5c06a.mp4

Hope this helps. Thank you

**Comment by alexandar-mitsev** [comment]
Hi @anuvenkatesh1 ,

Thanks for clarifying. I can also confirm the behaviour is the same for Jaws 2021.

@SAP/ui5-webcomponents-topic-rl , it is a reproducible issue with virtual cursor and navigation with CTRL-ALT-arrows inside the table. Please check.

Regards,
Alexandar Mitsev

**Comment by niyap** [comment]
Hello @2bno1,

Thanks for sharing your findings!

In ОpenUI5 the expected behaviour is achieved via “headers” attribute which has a reference to the column as a value. Setting IDRef relationships is currently not possible due to the nature of the custom elements and their shadow DOM and that is the reason to not be able to provide a reference to the column, when a table cell is accessed. You can find more information regarding this limitation here : https://sap.github.io/ui5-webcomponents/master/playground/advanced/accessibility/

 Kind Regards,
Niya

**Comment by 2bno1** [comment]
@niyap Please take a look at the comments above.
